defaults/main.yml

```diff
@@ -1 +1,33 @@
 ---
+
+# The database icinga2 will export it's configuration to.
+# default is "icinga2"
+icinga2_web_database_name: "icinga2"
+
+# The database user icinga2 will export it's configuration to.
+# default is "icinga2"
+icinga2_web_database_user: "icinga2"
+
+# The database password icinga2 will export it's configuration to.
+# default is "password"
+icinga2_web_database_pass: "password"
+
+# The database host icinga2 will export it's configuration to.
+# default is "localhost"
+icinga2_web_database_host: "127.0.0.1"
+
+# The database port icinga2 will export it's configuration to.
+# default is "localhost"
+icinga2_web_database_port: "3306"
+
+icinga2_master_database_name: "icinga2"
+icinga2_master_database_host: "127.0.0.1"
+icinga2_master_database_port: "3306"
+icinga2_master_database_user: "icinga2"
+icinga2_master_database_pass: "password"
```

Comment:
Always prefix all vars with the role name...

Reply:
I'll probably move this to the icinga2_master role, this was pushed by accident. Thanks for pointing that out!